reark/src/main/java/io/reark/reark/data/stores/cores/ContentProviderStoreCoreBase.java

```diff
@@ -166,12 +182,43 @@ public static Handler createHandler(@NonNull final String name) {
     }
 
     @NonNull
-    private Observable<ContentProviderOperation> createOperation(@NonNull final U item, @NonNull final Uri uri) {
+    private Observable<CoreOperation> createCoreOperation(@NonNull final CoreValue<U> value) {
+        switch (value.type()) {
+            case UPDATE:
```

Comment:
The return values in delete and the update are the same, just have `case UPDATE:` and `case DELETE:` one after another with one return line